actions/rssiActions.js

```diff
@@ -152,7 +152,7 @@ export function open(serialPort) {
 
 export function close() {
     return dispatch => {
-        if (port && port.isOpen()) {
+        if (port && port.isOpen && (typeof (port.isOpen) !== 'function' || port.isOpen())) {
```

Comment:
Though correct, it's not grouped logically. I'd suggest a condition that doesn't make you stop reading:
`port && (typeof (port.isOpen) === 'function' ? port.isOpen() : port.isOpen`)